taxonomies/categories.txt

```diff
@@ -19346,16 +19609,40 @@ fr:Jelly beans
 nl:Jelly beans
 zh:软心豆粒糖
 
+<en:Gelified candies
+en:Jelly candy
+fr:Bonbon gélifié
+ciqual_food_code:en:31060
+ciqual_food_name:en:Jelly candy
+ciqual_food_name:fr:Bonbon gélifié
+
 <en:Candies
-en:Lollipops
+en:Lollipops, Lollipop
 de:Dauerlutscher
 es:Piruletas
 fi:tikkarit
-fr:Sucettes
+fr:Sucettes, Sucette
```

Comment:
Simple singulars / plurals in French are handled automatically (when it's just adding or removing an s)

Reply:
I wasn't aware of that. Thanks!